### ISSUE TITLE
Create customise your setup page

### ISSUE THREAD

**Issue by O1LER** [BeamMP/Docs#53]
Create

https://docs.beammp.com/game/tailoring/

from scratch

**Comment by O1LER** [comment]
What even is there to customise? The launcher-to-game port? Or would this be about the ingame options that MP adds?

**Comment by ELKER54** [comment]
Yeah, not to sure about this. I guess maybe like user data locations, but otherwise not sure. Maybe @Starystars67 or another Mgmt person can give some advice

**Comment by O1LER** [comment]
Add descriptions for each ingame multiplayer option
